### PR TITLE
Added support for use_frameworks

### PR DESCRIPTION
**Pull request by azubala**
JIRA: https://observatory.atlassian.net/browse/TUN-3050

What's up:

- renamed pod from `25519` to `Curve25519` this way Cocoapods does not create framework with name `_25519` and we can safely import it’s headers
- added Fastlane setup
- added Bundler setup
- bumped version to `3.0.0` since the change of the pod name is a breaking API change, old imports won’t work

### CODE REVIEW

File: Curve25519.podspec [hunk 1]
```diff
@@ -11,9 +11,9 @@ Pod::Spec.new do |spec|
     Curve25519 is a fast and secure curve used for key agreement. Unfortunately, it does not support signing out of the box. This pod translates the point curves to do ed25519 signing with curve25519 keys.
   DESC
 
-  spec.source       = { :git => 'https://github.com/FredericJacobs/25519.git', :tag => "#{spec.version}" }
+  spec.source       = { :git => 'https://github.com/ynd-consult-ug/25519', :tag => "#{spec.version}" }
```
Comment (lukewar) [inline Curve25519.podspec]: Replace `#{spec.version}` with `spec.version.to_s`

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'cocoapods', '~> 1.2.0'
+gem 'fastlane'
```
Comment (lukewar) [inline Gemfile]: why not locking fastalne version?

File: fastlane/Fastfile [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+podspec_path = "Curve25519.podspec"
+ynd_pods_spec_repo = {
+  :name => "ynd",
+  :address => "git@bitbucket.org:ynd-consult-gmbh/cocoapods-spec-repo.git"
+}
+
+before_all do  
+  if git_tag_exists(tag: version_get_podspec)
+    UI.user_error!("Version #{pod_version} already exists! Make sure to update!")
```
Comment (lukewar) [inline fastlane/Fastfile]: It will fail. Extract `pod_version  = version_get_podspec`

File: Curve25519.podspec (hunk 1 above)
Comment (azubala) [inline Curve25519.podspec]: What's the advantage? In most podspecs I've seen ruby `#{ }` notation for converting variable to string

File: Gemfile (hunk 2 above)
Comment (azubala) [inline Gemfile]: Copy-paste fail :)

File: fastlane/Fastfile (hunk 3 above)
Comment (azubala) [inline fastlane/Fastfile]: Copy-paste fail :)

**Comment by azubala** [comment]
Added CR adjustments, please review.

**Pull request merged**